about.html

```diff
@@ -13,19 +13,12 @@
 </head>
 
 <body>
-  <!-- Mobile Nav -->
```

Comment:
I see you've deleted the mobile navigation, is there any reason for that?

The site also works on mobile, and it's important they have their own navigation bar

Reply:
sorry i had to clarify
i saw 2 home , about and contact in the **ABOUT** and **CONTACT** Pages
in the index.html page this issue does not appear for me